README.md

```diff
@@ -31,27 +31,32 @@ git clone git@github.com:openjdk/mobile.git
 ```
 
 ### Configure
-Modify the configure script below so that it has the correct paths to your JDK24 directory,
-the unzipped 'support' directory, and the correct location of the iPhoneOS.platform.
+Modify the configure script below so that it has the correct paths to the unzipped 'support'
+directory, and the correct location of the iPhoneOS.platform.
 
 ```
 sh configure \
---with-conf-name=zero-ios-aarch64 \
 --disable-warnings-as-errors \
 --openjdk-target=aarch64-macos-ios \
---with-boot-jdk=/Users/abhinay/.sdkman/candidates/java/24.ea.29-open \
---with-jvm-variants=zero \
 --with-libffi-include=<support-dir-absolute-path>/libffi/include \
 --with-libffi-lib=<support-dir-absolute-path>/libffi/libs \
 --with-cups-include=<support-dir-absolute-path>/cups-2.3.6
 --with-sysroot=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk \
 ```
 
+If you need to tell configure the path of the boot JDK, or if configure fails with an error saying
+it can't find the boot JDK, for instance if you downloaded a JDK in compressed archive form rather
```

Comment:
minor: change "the boot JDK" to "a boot JDK"

Reply:
I'll change this as soon as possible.